README.md

```diff
@@ -6,13 +6,13 @@ You can send events to Forwardlytics, and it will forward it to many services li
 
```

Comment:
You forgot the first line of the readme
![](https://i.ytimg.com/vi/Yt41V1Lt1IU/maxresdefault.jpg)